src/app/services/property-detail/property-detail.service.ts

```diff
@@ -3,15 +3,26 @@ import { Observable } from 'rxjs';
 import { HttpService } from 'src/app/services/http.service';
 
 @Injectable({
-    providedIn: 'root'
+  providedIn: 'root'
 })
 export class PropertyDetailService {
-    constructor(
-        private http: HttpService) {
-        }
-    getProperty(slug): Observable<any> {
+  isDisplayedModal: boolean = false;
+  constructor(private http: HttpService) {}
 
-        const endpoint = `/properties/${slug}`;   
-        return this.http.getRequestWithParams(endpoint);
-    }
+  getProperty(slug): Observable<any> {
+    const endpoint = `/properties/${slug}`;
+    return this.http.getRequestWithParams(endpoint);
+  }
+
+  display_Modal() {
```

Comment:
@MBenedicte you could simplify this code to `this.isDisplayedModal = this.isDisplayedModal?false:true`

Reply:
worked in this @alexxsanya